guides/common/modules/proc_disabling-dns-dhcp-tftp-for-unmanaged-networks.adoc

```diff
@@ -15,9 +19,11 @@ However, {Project} does not remove the back-end services on the operating system
 --foreman-proxy-tftp false
 ----
 
-. In the {ProjectWebUI}, navigate to *Infrastructure* > *Subnets* and select a subnet.
+. For every subnet where the {SmartProxy} is set as a DHCP, DNS, or TFTP proxy, disable the proxies:
+
+.. In the {ProjectWebUI}, navigate to *Infrastructure* > *Subnets* and select a subnet.
```

Comment:
```suggestion
.. In the {ProjectWebUI}, navigate to *Infrastructure* > *Subnets*.
.. Select a subnet.
```

suggestion (non-blocking): We generally split "navigate to X > Y" and "select Z" into two lines.